### PR TITLE
Add an option to copy symlinks

### PR DESCRIPTION
**Pull request by OlliV**

### CODE REVIEW

File: bin/now-deploy.js [hunk 1]
```diff
@@ -90,6 +92,7 @@ const help = () => {
     -f, --force               Force a new deployment even if nothing has changed
     -t ${chalk.underline('TOKEN')}, --token=${chalk.underline('TOKEN')}   Login token
     -L, --login               Configure login
+    -l, --links               Copy symlinks as symlinks
```
Comment (leo) [inline bin/now-deploy.js]: Let's make this a little more explicit:

> Don't resolve symlinks

**Review by leo — CHANGES_REQUESTED** [review]

File: bin/now-deploy.js (hunk 1 above)
Comment (OlliV) [reply bin/now-deploy.js]: Should we also add an option to ignore permissions or file mode completely?

Comment (rauchg) [reply bin/now-deploy.js]: Is it actually _not resolving symlinks_ ? I do agree that it needs re-wording, but not sure it's that one.

**Comment by rauchg** [comment]
What's the spirit of this option?

**Comment by OlliV** [comment]
@rauchg Currently it's not possible to use symlinks (with now-cli) in deployments because we always copy the contents of the file. So if you have a symlink to a file you just get two copies of the file inside the deployment instead of a symlink. With this option the user can opt-in for real symlinks.

File: bin/now-deploy.js (hunk 1 above)
Comment (OlliV) [reply bin/now-deploy.js]: btw. that's the same wording as rsync is using 🙂

**Pull request merged**